fili-wikipedia-example/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh 
+
+export HOSTIP="$(resolveip -s $HOSTNAME)" 
+/usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf 
```

Comment:
Before you were backgrounding this process and I'm guessing you want to do the same here 

```bash
/usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf &
```

It looks like it waits here and never tries to start up fili